python/datasources/cawp_time.py

```diff
@@ -555,20 +555,25 @@ def get_state_leg_totals_df():
     state_dfs = []
     for fips, id in FIPS_TO_STATE_TABLE_MAP.items():
         state_df = gcs_to_bq_util.load_csv_as_df_from_web(get_stleg_url(id))
+
+        # remove weird chars from col headers
         state_df.columns = state_df.columns.str.replace(r'\W', '', regex=True)
 
+        # remove non-digits from buggy index/years column
+        state_df["10"] = state_df["10"].astype(str).replace(
```

Comment:
Why is the col name "10"?

Reply:
That's the buggy index/year column situation. This is what the source table looks like:
<img width="1251" alt="Screen Shot 2023-01-04 at 2 57 47 PM" src="https://user-images.githubusercontent.com/41567007/210657743-e3d0f521-a2b9-40d8-b22b-7fae001bc5f7.png">

Linked from the state leg. table on this page (and some of the other states) https://cawp.rutgers.edu/facts/state-state-information/arizona

Reply:
Cool, would you mind throwing a comment on there describing this?